python/lsst/rucio/register/rucio_interface.py

```diff
@@ -296,7 +301,13 @@ def register_as_replicas(self, dataset_id, dataset_refs) -> None:
         """
         bundles = []
         for dataset_ref in dataset_refs:
-            bundles.append(self._make_bundle(dataset_id, dataset_ref))
+            if type(dataset_ref) is list:
+                for dsr in dataset_ref:
+                    logging.debug(f"{self.butler.getURI(dsr)=}")
+                    logging.debug(f"{dsr.to_json()=}")
+                    bundles.append(self._make_bundle(dataset_id, dsr))
+            else:
+                bundles.append(self._make_bundle(dataset_id, dataset_ref))
```

Comment:
It looks like this package already depends on **lsst.utils**, so you may consider using [ensure_iterable()](https://pipelines.lsst.io/py-api/lsst.utils.iteration.ensure_iterable.html#lsst.utils.iteration.ensure_iterable) to eliminate the if statement entirely. But I'm leaving this decision up to you.

Reply:
going to leave this as is, at least for now.